lib/game.rb

```diff
@@ -0,0 +1,81 @@
+class Game
+	attr_accessor :board, :player_1, :player_2
+
+	WIN_COMBINATIONS = [
+    [0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8],
+    [0, 4, 8], [2, 4, 6]
+  ]
+
+
+  def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board = Board.new)
+  	@player_1 = player_1
+  	@player_2 = player_2
+  	@board = board
+    
+  end
+
+  def current_player
+  	if board.turn_count.even?
+  		player_1
+  	else
+  		player_2
+  	end
+  end
+
+  def full?
+  	if board.any?{|i| i != player_1 && i != player_2}
+  		false
+  	else
+  		true
+  	end		
+  end
+
+  def won?
+	WIN_COMBINATIONS.find do |combo|
+  		self.board.cells[combo[0]] == self.board.cells[combo[1]] && self.board.cells[combo[1]] == self.board.cells[combo[2]] && self.board.cells[combo[0]] != " "
+  	end
+  end
+
+  def draw?
+  	board.full? && !won?
+  end
+
+  def over?
+  	draw? || won?
+  end
+
+  def winner
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.